src/components/common/BcRegEntityDetails.vue

```diff
@@ -0,0 +1,310 @@
+<template>
+  <div>
+    <template v-if="isBenefit">
```

Comment:
I'm sort of OK with having the HTML for all entity types in this one file, but did you consider having separate sub-components for each type?

You could use dynamic components here.
Example: https://github.com/bcgov/business-filings-ui/blob/fafe0c3af7da84fa91f3677f785cfe5c7f0adb8b/src/components/Dashboard/FilingHistoryList.vue#L46

You may have already tried that and your current implementation may be cleaner. If so, say so :)

Reply:
Would consider it, but this is mostly a 1 off?